src/components/info/Notice/Notice.style.ts

```diff
@@ -38,15 +40,16 @@ export const NoticeChildren = styled(Text)`
   color: white;
 `;
 
-function padding({ icon = null, pill = null }) {
+function padding({ icon = null, onClose = null, pill = null }) {
```

Comment:
`onClose` is not being passed to this function and as such the value of `dismissPad` will always be `0`.